### PR TITLE
Tree relabeler requirements fix

### PR DESCRIPTION
**Pull request by Kurt-Hanel**
An updated version of the tree re-labeler wrapper that works on the latest release of galaxy. The main change is the deletion of the `requirements` section and the `tool_dependencies.xml` file -- both called for the `nmltoolkit` package, which could not be found.

### CODE REVIEW

**Comment by ericenns** [comment]
@Takadonet will want you to add perl depencies via conda, as the libraries required will not be available on all of the nodes that the job might run on by default. Take a look at https://github.com/phac-nml/galaxy_tools/blob/master/tools/fasta2bed/fasta_to_bed.xml for an example.

Also it would be a good idea to add tests to the wrapper, to ensure that the fix you applied actually fixes the issue.

File: tools/tree_relabeler/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The information in the tab file is inserted into the tree file so the new information will appear on the tip labels.
+
+Input
+Newick format tree
+
+Tab delimted file
+
+Desired delimiter for updated tip labels
+name: Tree Relabeler
+owner: PHAC/NML
```
Comment (ericenns) [inline tools/tree_relabeler/.shed.yml]: owner should be `nml` which is what we use on the toolshed

File: tools/tree_relabeler/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+categories:
+- Text Manipulation 
+description: Relabels the tips of a newick formatted tree. 
+long_description: |
```
Comment (ericenns) [inline tools/tree_relabeler/.shed.yml]: when using `|` to indicate a block in yaml, you need to indent all the following lines of the block

File: tools/tree_relabeler/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+categories:
+- Text Manipulation
```
Comment (ericenns) [inline tools/tree_relabeler/.shed.yml]: lists in yaml need to be indented

**Comment by ericenns** [comment]
:+1 @Takadonet should do a final review before merge though

**Comment by Kurt-Hanel** [comment]
Sounds good, thanks for the help @ericenns!

**Pull request merged**